raiden/network/transport/matrix/transport.py

```diff
@@ -510,6 +511,11 @@ def start_health_check(self, node_address):
             # representing the target node
             self._address_mgr.refresh_address_presence(node_address)
 
+            # if not is_peer_reachable:
```

Comment:
Why is this block of code commented? 

The `force_user_presence ` logic was removed